src/Workspaces/CoreTest/ChecksumTests.cs

```diff
@@ -6,67 +6,77 @@
 using System.Linq;
 using Xunit;
 
-namespace Microsoft.CodeAnalysis.UnitTests
+namespace Microsoft.CodeAnalysis.UnitTests;
+
+public class ChecksumTests
 {
-    public class ChecksumTests
+    [Fact]
+    public void ValidateChecksumFromSpanSameAsChecksumFromBytes1()
     {
-        [Fact]
-        public void ValidateChecksumFromSpanSameAsChecksumFromBytes1()
-        {
-            var checksum1 = Checksum.Create("Goo");
-            var checksum2 = Checksum.Create("Bar");
+        var checksum1 = Checksum.Create("Goo");
+        var checksum2 = Checksum.Create("Bar");
 
-            var checksumA = Checksum.Create(checksum1, checksum2);
+        var checksumA = Checksum.Create(checksum1, checksum2);
 
-            // Running this test on multiple target frameworks with the same expectation ensures the results match
-            Assert.Equal(Checksum.FromBase64String("N30m5jwVeMZzWpy9cbQbtSYHoXU="), checksumA);
+        // Running this test on multiple target frameworks with the same expectation ensures the results match
+        Assert.Equal(Checksum.FromBase64String("RRbspG+E4ziBC5hOWyrfCQ=="), checksumA);
```

Comment:
These changes are expected.  We are legit changing our hash from sha256 to xxhash128, so the results are different.  THe important thing is that we get the same result on netfx and netcore (So our two servers can communicate just fine).